tools/tsconfig/base.json

```diff
@@ -9,7 +9,7 @@
     "forceConsistentCasingInFileNames": true,
     "inlineSources": false,
     "isolatedModules": true,
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
```

Comment:
Does this impact our dist files in any way?

Reply:
nope, just the typechecking of how typescript resolves modules. Doesn't impact the bundle resolution strategy